src/OpenLoco/Widget.cpp

```diff
@@ -232,70 +232,69 @@ namespace OpenLoco::Ui
 
     void Widget::sub_4CADE8(Gfx::Context* context, const Window* window, uint8_t colour, bool enabled, bool disabled, bool activated)
     {
-        int16_t x = left + window->x;
-        int16_t y = top + window->y;
-        uint32_t image = image;
-
+        int16_t x_place_for_image = left + window->x;
+        int16_t y_place_for_image = top + window->y;
+        uint32_t new_image = image;
```

Comment:
perhaps `imageId` would be a better name. Its not really a new image its just the current image that we want to either remove the colouring or make the activated version.